pkgs/by-name/li/libossp_uuid/package.nix

```diff
@@ -13,9 +13,9 @@ stdenv.mkDerivation rec {
     sha256 = "11a615225baa5f8bb686824423f50e4427acd3f70d394765bdff32801f0fd5b0";
   };
 
-  configureFlags = [
-    "ac_cv_va_copy=yes"
-  ] ++ lib.optional stdenv.hostPlatform.isFreeBSD "--with-pic";
+  configureFlags =
+    lib.optional (stdenv.buildPlatform != stdenv.hostPlatform) "ac_cv_va_copy=C99"
```

Comment:
Let’s just set this unconditionally. From reading `aclocal.m4`, it seems like any compiler that isn’t from the stone age will be `C99` here, and anything that doesn’t support basic C99 won’t get through Nixpkgs bootstrap anyway. Best to avoid cross‐only conditionals when possible.